docs/docs/howto/clusterconnectivity/aiven-kafka-connect-cluster-ssl-protocol.rst

```diff
@@ -26,8 +29,9 @@ Follow the steps below to configure and connect an Aiven for Apache Kafka Connec
 4. Click **Save**. 
 5. Add the cluster to the preferred environment. Click **Environments** from the **Environments** drop-down menu.
 6. In the **Kafka Connect Environments** section, click **Add Environment** and enter the details to add your schema registry environment. Click **Save**. 
-7. Open the ``application.properties`` file for `core`` (klaw/core/src/main/resources) and `cluster-api` (klaw/cluster-api/src/main/resources) modules. 
-8. Configure the SSL properties to connect to Schema Registry cluster by copying and editing the following lines:
+7. Open the ``application.properties`` file for `core`` (klaw/core/src/main/resources) and `cluster-api` (klaw/cluster-api/src/main/resources) modules.
```

Comment:
May be the application properties only from cluster-api is needed here

Reply:
The SSL properties if configured on the cluster api must be configured on core or it won't be able to communicate over https to the cluster api.